Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -924,6 +924,11 @@ class BrowserViewController: UIViewController {
         screenshotHelper.takePendingScreenshots(tabManager.allTabs)
 
         super.viewDidAppear(animated)
+        
+        // NTP Education Load after onboarding screen
+        if showNTPEducation().isEnabled, let url = showNTPEducation().url {
+            tabManager.selectedTab?.loadRequest(PrivilegedRequest(url: url) as URLRequest)
```

Comment:
Two things here:
1. Isn't this going to be opened at every app launch? Shouldn't we store a preference once we showed it once
2. Should we show it to existing users?

Reply:
Had a logic change to only new user and add it to completion of onboarding. 
In addition onboarding logic is moved to a new extension